pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -315,7 +313,7 @@ func (c *InstallerController) manageInstallationPods(ctx context.Context, operat
 			// if we make a change to this status, we want to write it out to the API before we commence work on the next node.
 			// it's an extra write/read, but it makes the state debuggable from outside this process
 			if !equality.Semantic.DeepEqual(newCurrNodeState, currNodeState) {
-				klog.Infof("%q moving to %v because %s", currNodeState.NodeName, spew.Sdump(*newCurrNodeState), reason)
+				klog.Infof("%q is moving to %q with Target Revision: %d and Current Revision: %d because %s and reached the Expected Revision %d", currNodeState.NodeName, newCurrNodeState.NodeName, newCurrNodeState.TargetRevision, newCurrNodeState.CurrentRevision, reason, newCurrNodeState.CurrentRevision)
```

Comment:
the node name is constant. It's the key of the list / set.

Reply:
are we sure the other fields an never changed? lastFailedRevision or what we have there in addition.